### DISCUSSION TITLE
Gnu-Tools and a couples Ideas and Suggestions (clink looks not bad :) )

### DISCUSSION THREAD

**Discussion by blackcrack** [chrisant996/clink#354] [Ideas]
Hi,

a idea/suggestion, why do you not add Gnu-Tools in a subdir at the Window-System, 
by side of System32 and SysWOW64, GnuTools add at the Systempath for users  or so.. 
;%windir%/gnutools
 and put there all Mingw Gnutools into.. therewith worls also "ls", "mkdir" and "rm" from Linux btw. from gnutools..
for Win32:
https://gnuwin32.sourceforge.net/packages.html
https://sourceforge.net/projects/gnuwin32/files/
for 64Bit WinNT
https://sourceforge.net/projects/mingw-w64/files/mingw-w64/mingw-w64-release/

and have a try to make the starter as exe file who works also in sambafs 
\\Linuxserver\dir\clinkv1.3.44\clink.bat
\\Linuxserver\dir\clinkv1.3.44\clinkstart.exe

then is it also possible to brows over netfs and start on a smbfs.. and works also in Wine ..
for have it in a Sub-directory in the net and makes more sense if runs a commandointerpreter who is linuxlike.

maybe make , if starts, the actual directory as temporal add-on to the path.variable so long clink runs.

i be since more as 25 years in the net.. and working over unc-path with an not compatible script-file, makes no sense ..
because it works not really.. therefore an c or c++ / NetFx compiled file (and don't use and Bat2exe this works also not..)
all script languages be not really the best imho, then an really binary language. An binary language is independent.
you should use an SMBFs compatible binary who is translated directly in binary code.

my prompt looks like so :  
```
Microsoft Windows [Version 6.1.7601]
Copyright (c) 2009 Microsoft Corporation. Alle Rechte vorbehalten.

\\Linuxserver\Windows\0=----Arbeiten--Bastelleien-Store-Arbeiten von mir--------
0\00=-Reactos-Installer-WS=-00\GnuWin>_

```
this works with an reg-key who disable the warning about unc-path..
have i tweak with  :  
http://www.winfaq.de/download_rsw.htm
-=>take the portable (i know, it's old, but the Tweaks works mostly in Win11 too as Win10)

so, just for suggestion and cc"IDEA" as put my 2 Cent's in ;) 

best

**Comment by chrisant996** [comment]
Hi, thanks for starting the conversation.

The idea is fine, but doesn't really fit as something to build in as part of Clink.
For example, I don't use any of those things, and I have no need or interest in using them.

Maybe you could build an installable project that installs those things, to make it easy for people who want them.
I could add a link in the Clink documentation to help people find the project.
